sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -1223,7 +1223,7 @@ object Radar {
     }
 
     /**
-     * Geocodes the device's current IP address, converting IP address to country.
+     * Geocodes the device's current IP address, converting IP address to RadarAddress.
```

Comment:
@kochis Can you say `converting IP address to partial address` to match the API docs?